pkgs/data/icons/beauty-line-icon-theme/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, gtk3, gnome-icon-theme, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  pname   = "BeautyLine";
+  version = "0.0.1";
+
+  src = builtins.fetchTarball {
+    name   = "${pname}-${version}";
+    url    = "https://github.com/gvolpe/BeautyLine/releases/download/${version}/BeautyLine.tar.gz";
+    sha256 = "0s9zpxh8s93awnkw30xnxmpnqsw6wx1fvyvjh33fs3r40vv7lxgg";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
+
+  propagatedBuildInputs = [ gnome-icon-theme hicolor-icon-theme ];
```

Comment:
Please, add the other inherited themes (see `index.theme`) as propagated build inputs
```
Inherits=breeze,gnome,ubuntu-mono,Mint-X,elementary,hicolor
```
- breeze
- ubuntu-mono
- mint-x
- elementary

Reply:
@romildo any ideas what Nix package corresponds to `ubuntu-mono`? It's the only one missing:

```
Symlinking parent icon themes...
  theme: BeautyLine
    parent: breeze      -> /nix/store/mgs1gzdsdwd4646apylcbhbc574p7rxd-breeze-icons-5.78.0/share/icons/breeze
    parent: gnome       -> /nix/store/n1kbpz5g9nqi9bxphy2b31cnzswrh7g8-gnome-icon-theme-3.12.0/share/icons/gnome
    parent: ubuntu-mono ->
    parent: Mint-X      -> /nix/store/mcjmxinqv9qbg2rw28hzc5rja2hcgydz-mint-x-icons-1.5.5/share/icons/Mint-X
    parent: elementary  -> /nix/store/723qlccpwlfj4cql93r7qk0q9sy3yy4i-elementary-icon-theme-5.3.1/share/icons/elementary
    parent: hicolor     -> /nix/store/flx78mblhi7nlryyrzgwg42giw99sb7s-hicolor-icon-theme-0.17/share/icons/hicolor
```

I tried `ubuntu-themes` without success.

Reply:
`ubuntu-themes` provides `ubuntu-mono-light` and `ubuntu-mono-dark`, but not `ubuntu-mono` :)

I think you can add a comment explaining that this parent theme is missing, and do not worry too much about it now. If eventually it becomes packaged, one can go back and update this package to include it.

Parent themes are useful when an icon is not available in the primary theme. It is then searched in the parent themes, in the order they appear in `index.theme`. This theme already have plenty of parent themes.

Reply:
Awesome, thanks a lot for helping out, I added a comment :)